openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -926,14 +926,13 @@ If the Credential Response is not encrypted, the media type of the response MUST
 
 The following parameters are used in the JSON-encoded Credential Response body:
 
-* `format`: REQUIRED. String denoting the format of the issued Credential.
 * `credential`: OPTIONAL. Contains issued Credential. It MUST be present when `transaction_id` is not returned. It MAY be a string or an object, depending on the Credential format. See (#format_profiles) for the Credential format specific encoding requirements.
 * `transaction_id`: OPTIONAL. String identifying a Deferred Issuance transaction. This claim is contained in the response if the Credential Issuer was unable to immediately issue the Credential. The value is subsequently used to obtain the respective Credential with the Deferred Credential Endpoint (see (#deferred-credential-issuance)). It MUST be present when the `credential` parameter is not returned. It MUST be invalidated after the Credential for which it was meant has been obtained by the Wallet.
 * `c_nonce`: OPTIONAL. String containing a nonce to be used to create a proof of possession of key material when requesting a Credential (see (#credential_request)). When received, the Wallet MUST use this nonce value for its subsequent Credential Requests until the Credential Issuer provides a fresh nonce.
 * `c_nonce_expires_in`: OPTIONAL. Number denoting the lifetime in seconds of the `c_nonce`.
 * `notification_id`: OPTIONAL. String identifying an issued Credential that the Wallet includes in the Notification Request as defined in (#notification). This parameter MUST NOT be present if `credential` parameter is not present.
 
-The value of the `format` parameter determines the Credential format and encoding of the credential in the Credential Response. Details are defined in the Credential Format Profiles in (#format_profiles).
+The format and encoding of the `credential` value is determined by the Credential format specified in the Credential Request. Details are defined in the Credential Format Profiles in (#format_profiles).
 
 Credential formats expressed as binary data MUST be base64url-encoded and returned as a string.
```

Comment:
```suggestion
The format of the `credential` value is determined by the Credential format specified in the Credential Request. 

The encoding of the Credential in the `credential` value depends on the Credential Format Profiles defined in (#format_profiles). Credential formats expressed as binary data MUST be base64url-encoded and returned as a string.
```

Reply:
Is this an acceptable compromise? @Sakurann @awoie

Reply:
To me that works. 

I think your suggestion makes it more clear that encoding refers to the encoding of the `credential` value and does not refer to the encoding of the credential itself. For instance, mdocs have a native CBOR encoding while being encoded as base64url in the `credential` value.

Reply:
here is my suggestion

```suggestion
The format of the Credential in the Credential Response is determined by the value of the `format` parameter specified in the Credential Request.

The encoding of the Credential returned in the `credential` parameter depends on the Credential Format. Credential formats expressed as binary data MUST be base64url-encoded and returned as a string.

More details such as Credential Format identifiers are defined in the Credential Format Profiles in (#format_profiles). 
```